src/malware_prediction/main_model.py

```diff
@@ -21,21 +21,16 @@
 # TRAIN_ENCODED_CSV_PATH = f"./data/train/train_encoded.csv"
 # TEST_ENCODED_CSV_PATH = f"./data/test/test_encoded.csv"
 
-ROWS = 0
-MESSAGE = """
-            --- {hostname} ---
-                                                          
-            --- {run_time} S ---"""
+ROWS = DataLoader(MalwareDataset.load(TRAIN_ENCODED_CSV_PATH), batch_size=1).dataset.rows
```

Comment:
Wir führen dreimal MalwareDataset.load(TRAIN_ENCODED_CSV_PATH) in diesem File aus, ich glaube das ist nicht sonderlich resourcensparend